go/chat/attachments/preprocess.go

```diff
@@ -201,7 +201,12 @@ func PreprocessAsset(ctx context.Context, log utils.DebugLabeler, filename strin
 	}
 	// MIME type detection failed us, try using an extension map
 	if p.ContentType == "application/octet-stream" {
-		if typ, ok := mimeTypes[strings.ToLower(filename)]; ok {
+		var ext string
```

Comment:
https://golang.org/pkg/path/#Ext should do this...

Reply:
Ah good call of course, will change!